zeppelin-server/src/main/java/org/apache/zeppelin/types/InterpreterSettingsList.java

```diff
@@ -24,14 +24,14 @@
 /**
  * InterpreterSetting information for binding
  */
-public class InterpreterSettingListForNoteBind {
+public class InterpreterSettingsList {
   private String id;
   private String name;
   private boolean selected;
   private List<InterpreterInfo> interpreters;
 
-  public InterpreterSettingListForNoteBind(String id, String name,
-      List<InterpreterInfo> interpreters, boolean selected) {
+  public InterpreterSettingsList(String id, String name,
+                                 List<InterpreterInfo> interpreters, boolean selected) {
```

Comment:
Vertial align is not recommended. Could you please revert this into original one?